src/main/resources/resources/logisim/strings/gui/gui_pt.properties

```
@@ -274,8 +274,8 @@ simulateEnableTicksTip = Habilitar pulsos de clock
 simulateRunTip = Executar o simulador continuamente
 simulateStepTip = Propagação de pulso únitário do sinal através do circuito
 simulateStopTip = Simulador de parada
-simulateTickFullTip = Tick relógio um ciclo completo
-simulateTickHalfTip = Marcador de relógio um meio ciclo
+simulateTickFullTip = Tick clock um ciclo completo
```

Can you please research if `clock tick` has no native translation in your language? It'd bet there's one.

---

I was thinking about that one for a bit. This is a tricky one because for majority things related to electronic and hardware we already use Clock and Tick, but the context can be optimized.

Gonna research and think a little more about this one

---

If we use "Relógio" which is the literal translation of "Clock", it would sound ironically out of context, even if its right. We say "O clock do processador é 3.0ghz" as "The processor clock is 3.0ghz". Maybe we can put it like this?

```suggestion
simulateTickFullTip = Tick do clock em um ciclo completo
simulateTickHalfTip = Tick do clock em meio ciclo
```

---

The only comment here I can have is that according to SI units it's `Hz` not `hz`. Can you please correct that so we follow SI units?